### PR TITLE
brotli: add page

### PR DESCRIPTION
**Pull request by Sosthene-Guedon**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/brotli.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Brotli
+
+> Compress/uncompress files with brotli compression.
+
```
Comment (bl-ue) [inline pages/common/brotli.md:4]: We like to add a more information link when possible:
```suggestion
> More information: <https://github.com/google/brotli>.

```

**Review by schneiderl — APPROVED** [review]

File: pages/common/brotli.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+`brotli {{file.ext}} -o {{compressed_file.ext.br}}`
+
+- Decompress a brotli file specifying the output filename:
+
+`brotli -d {{compressed_file.ext.br}} -o {{file.ext}}`
+
+- Specify the compression level. 1=Fastest (Worst), 11=Slowest (Best):
+
+`brotli -q 11 {{file.ext}} -o {{compressed_file.ext.br}}`
```
Comment (sbrl) [inline pages/common/brotli.md:24]: Missing token syntax?
```suggestion
`brotli -q {{11}} {{file.ext}} -o {{compressed_file.ext.br}}`
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left 1 teensy comment below for you to review, and then we'll be good to merge :D

File: pages/common/brotli.md (hunk 2 above)
Comment (Sosthene-Guedon) [reply pages/common/brotli.md]: Applied in b695c43ca59c4fe4af43acf45f1b63f1ca845110

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**